backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -58,61 +50,69 @@ func New(cfg *any.Any, _ *zap.Logger, _ tally.Scope) (service.Service, error) {
 }
 
 func (fs *experimentStore) CreateExperiments(ctx context.Context, experiments []*experimentation.Experiment) error {
-	psql := sq.StatementBuilder.PlaceholderFormat(sq.Dollar)
-	builder := psql.Insert(experimentsTableName).Columns(experimentColumns...)
+	// This API call will eventually be broken into 2 separate calls:
+	// 1) creating the config
+	// 2) starting a new experiment with the config
+
+	// All experiments are created in a single transaction
+	tx, err := fs.db.Begin()
+	if err != nil {
+		return err
+	}
 
 	for _, experiment := range experiments {
-		marshaler := jsonpb.Marshaler{}
-		buf := &bytes.Buffer{}
-		err := marshaler.Marshal(buf, experiment.GetConfig())
+		// Step 1) create the config
+		configID := id.NewID()
+
+		configJson, err := marshalConfig(experiment)
```

Comment:
i think this should be experiment.Config, no? in fact it seems like this method should take an array of configs, not an array of experiments, as experiments already have an id.

if i'm reading correctly experiments created here would look like this in the db:
```yaml
id: 123
details:
  id: null
  config:
    @type: foo
    bar: baz
```

Reply:
better with change, but can you address question of whether or not to change interface to array of configs vs array of experiments?

Reply:
It needs to be an array of experiments. I set the experiment.Id for each experiment on line 96.

Reply:
still doesn't quite track for me because the user input will allow an id if we gen docs for the API or someone looks at the protos but i think it's trivial for now